src/views/DidDowngrade/DidDowngrade.tsx

```diff
@@ -61,13 +63,29 @@ function useCosts(
       }
 
       setFee(await getFee(did));
-      setDeposit(await getDeposit());
+      setDepositDid(await getDepositDid(did));
+      setDepositWeb3Name(await getDepositWeb3Name(did));
     })();
   }, [did]);
 
+  const deposit = useMemo(() => {
```

Comment:
I assume it didn’t work out then to improve this memo?

Reply:
I forgot what your initial suggestion was, maybe you can remind me? I couldn't come up with a good solution.

Reply:
```js 
const didDeposit = ... ? 0 : ...;
const nameDeposit = ... ? 0 : ...;
return didDeposit.add(nameDeposit);